api/src/test/java/org/openmrs/module/metadatamapping/api/MetadataMappingServiceTest.java

```diff
@@ -856,7 +856,7 @@ public void getMetadataItem_shouldReturnNullIfTermDoesNotExist() {
 	@Test
 	@Verifies(value = "return unretired metadata items of unretired terms matching type", method = "getMetadataItems(Class,"
 	        + " String)")
-	public void getMetadataItems_shouldReturnUnretiredMetadataItemsOfUnretiredTermsMatchingType() {
+	public void getMetadataItems_shouldReturnMetadataItemsOfTermsMatchingType() {
```

Comment:
Please make sure the `Verifies` annotation value matches the test method name and also the `verifies` tags in the javadoc for `org.openmrs.module.metadatamapping.api.MetadataMappingService#getMetadataItems`.